apps/dashboard/app/(app)/workspace-navigations.tsx

```diff
@@ -121,7 +124,7 @@ export const createWorkspaceNavigation = (
       href: "/identities",
       label: "Identities",
       active: segments.at(0) === "identities",
-      hidden: !workspace.betaFeatures.identities,
```

Comment:
why?
this is very much intended to be hidden

Reply:
My bad. I had to enable it to make sure everything works.